### ISSUE TITLE
Using homebrew SDKs with UniSpy?

### ISSUE THREAD

**Issue by InternalLoss** [GameProgressive/UniSpySDK#30]
Wiki states that consoles may be dropped in future since it requires use of the real SDK - almost all of these platforms have their own homebrew SDK (360 does not - but OG Xbox does! not that GameSpy was used on that platform, but) that could be used instead?

**Comment by koujiangheng** [comment]
make sense

**Comment by arves100** [comment]
yes it could once I push a more sane structure that can allow to easily implement new platform, altrough the license of the SDK is sort of obscure atm so I'm not sure how many people would use it on homebrew sdks (considering that they would have to port it to them first)
